src/compiler/binder.ts

```diff
@@ -266,6 +266,9 @@ namespace ts {
                     Debug.assert(isWellKnownSymbolSyntactically(nameExpression));
                     return getPropertyNameForKnownSymbolName(idText((<PropertyAccessExpression>nameExpression).name));
                 }
+                if (isPrivateName(node)) {
+                    return nodePosToString(node) as __String;
+                }

```

Comment:
ignore this: it's a mistake and to be removed in https://github.com/bloomberg/TypeScript/pull/5. Actually has no effect, since there is no way the `node` can be a private name here anyway.